recipes/podofo/all/conanfile.py

```diff
@@ -35,26 +40,18 @@ class PodofoConan(ConanFile):
         "with_tiff": True,
         "with_png": True,
         "with_unistring": True,
+        "with_tools": False,
     }
 
-    generators = "cmake", "cmake_find_package"
-
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
-
-    @property
-    def _build_subfolder(self):
-        return "build_subfolder"
+    generators = "CMakeToolchain", "CMakeDeps", "VirtualRunEnv"
```

Comment:
```suggestion
```
- Do not duplicate generators. You will run the same function twice.
- VirtuaEnv is only needed when you have build requirements.

Reply:
Ok I had the same comment from [prince-chrismc](https://github.com/prince-chrismc) so I guess there is something I do not understand here (and that is present in most of the existing recipe), but ok I removed them.